lib/screens/v2/departure.ex

```diff
@@ -198,13 +198,12 @@ defmodule Screens.V2.Departure do
   def vehicle_status(_), do: nil
 
   defp crowding_data_relevant?(%Trip{id: trip_trip_id, stops: [first_stop | _]}, %Vehicle{
-         current_status: current_status,
          trip_id: vehicle_trip_id,
          stop_id: next_stop
        })
        when not is_nil(trip_trip_id) and not is_nil(vehicle_trip_id) do
     vehicle_on_prediction_trip? = trip_trip_id == vehicle_trip_id
-    vehicle_started_trip? = not (current_status == :in_transit_to and next_stop == first_stop)
+    vehicle_started_trip? = not (next_stop == first_stop)
```

Comment:
There is an existing `crowding_level/1` test case `"returns nil when the vehicle is in transit to the first stop"`. I don't know if the _content_ of this test needs to change exactly, but maybe the wording could change to remove the reference to a condition that doesn't apply anymore.

Reply:
Changed the name just to remove the In Transit part since the test still make sense!